DOCS/man/options.rst

```diff
@@ -5286,14 +5286,22 @@ them.
     ``mitchell``
         Mitchell-Netravali. The ``B`` and ``C`` parameters can be set with
         ``--scale-param1`` and ``--scale-param2``. This filter is very good at
-        downscaling. This is the default for ``--dscale``.
+        downscaling.
+
+    ``hermite``
+        Hermite spline. Similar to ``bicubic`` but with ``B`` set to ``0.0``.
+        This filter has the special property of having a support of radius 1.0,
+        making it very fast in comparison. This filter is very good at
+        downscaling, as well as at frame mixing (``--tscale``). This is the
+        default for ``--dscale``.
 
     ``catmull_rom``
         Catmull-Rom. A Cubic filter in the same vein as ``mitchell``, where
         the ``B`` and ``C`` parameters are ``0.0`` and ``0.5`` respectively.
         This filter is sharper than ``mitchell``, but it results in mild
         ringing. Like ``mitchell``, this filter is good at downscaling (see
-        ``--dscale``).
+        ``--dscale``), but it is also good at upscaling (similar to and
+        slightly faster than ``lanczos``).
```

Comment:
This is gross simplification. good/similar/faster are empty words. Lanczos is sharper and alias less. You made is sound like catmull_rom is faster and better.

Reply:
Yeah I wouldn't say it's similar, it has way different artifacts, maybe saying "similar sharpness" would be better. "good at upscaling" is technically true but I think Lanczos is way better for luma while catmull is better for chroma.

Reply:
I'm not sure catmull_rom can really be considered "good at upscaling" considering how aliased it is in comparison to other options, maybe the description of being "good at downscaling and upscaling" can just be removed? The filter is really just not particularly great compared to other options for both downscaling and upscaling IMO.